src/goto-instrument/contracts/contracts.h

```diff
@@ -220,331 +246,4 @@ class code_contractst
-
-  code_contractst &parent;
-  messaget log;
-  irep_idt fun_id;
-
-  // written by the child classes.
-  std::string memmap_name;
-  std::string requires_fn_name;
-  std::string ensures_fn_name;
-};
-
-class is_fresh_enforcet : public is_fresh_baset
-{
-public:
-  is_fresh_enforcet(
-    code_contractst &_parent,
-    messaget _log,
-    const irep_idt _fun_id);
-
-  virtual void create_declarations();
-
-protected:
-  virtual void create_requires_fn_call(goto_programt::targett &target);
-  virtual void create_ensures_fn_call(goto_programt::targett &target);
-};
-
-class is_fresh_replacet : public is_fresh_baset
-{
-public:
-  is_fresh_replacet(
-    code_contractst &_parent,
-    messaget _log,
-    const irep_idt _fun_id);
-
-  virtual void create_declarations();
-
-protected:
-  virtual void create_requires_fn_call(goto_programt::targett &target);
-  virtual void create_ensures_fn_call(goto_programt::targett &target);
-};
-
-/// Predicate to be used with the exprt::visit() function.  It
-/// will return the set of function calls within a goto program.
-class find_is_fresh_calls_visitort
-{
-public:
-  find_is_fresh_calls_visitort()
-  {
-  }
-
-  // \brief return the set of functions invoked by
-  // the call graph of this program.
-  std::set<goto_programt::targett> &is_fresh_calls();
-  void clear_set();
-  void operator()(goto_programt &prog);
-
-protected:
-  std::set<goto_programt::targett> function_set;
-};
-
-/// Predicate to be used with the exprt::visit() function. The function
-/// found_return_value() will return `true` iff this predicate is called on an
-/// expr that contains `__CPROVER_return_value`.
-class return_value_visitort : public const_expr_visitort
-{
-public:
-  return_value_visitort() : const_expr_visitort(), found(false)
-  {
-  }
-
-  // \brief Has this object been passed to exprt::visit() on an exprt whose
-  //        descendants contain __CPROVER_return_value?
-  bool found_return_value();
-  void operator()(const exprt &exp) override;
-
-protected:
-  bool found;
-};
-
-/// Predicate to be used with the exprt::visit() function. The function
-/// found_return_value() will return `true` iff this predicate is called on an
-/// expr that contains `__CPROVER_return_value`.
-class functions_in_scope_visitort
-{
-public:
-  functions_in_scope_visitort(
-    const goto_functionst &goto_functions,
-    messaget &log)
-    : goto_functions(goto_functions), log(log)
-  {
-  }
-
-  // \brief return the set of functions invoked by
-  // the call graph of this program.
-  std::set<irep_idt> &function_calls();
-  void operator()(const goto_programt &prog);
-
-protected:
-  const goto_functionst &goto_functions;
-  messaget &log;
-  std::set<irep_idt> function_set;
-};
-
-class function_binding_visitort : const_expr_visitort
-{
-public:
-  function_binding_visitort() : const_expr_visitort()
-  {
-  }
-
-  void operator()(const exprt &exp) override
-  {
-  }
-};
-
 #endif // CPROVER_GOTO_INSTRUMENT_CODE_CONTRACTS_H
```

Comment:
The header guard no longer matches the path.

```suggestion
#endif // CPROVER_GOTO_INSTRUMENT_CONTRACTS_CONTRACTS_H
```

Also update at the top of this file

Reply:
Fixed it.